build

```diff
@@ -4,7 +4,7 @@
 if [ -f /etc/os-release ]; then
     . /etc/os-release
     DISTRO=$NAME
-    if [ "$DISTRO"=="Arch Linux" ]; then
+    if [[ "$DISTRO"=="Arch Linux" && "$DISTRO"=="Antergos Linux" && "$DISTRO"=="Manjaro Linux" ]]; then
```

Comment:
this makes no sense, you wanted to use or here...

Reply:
Whoops